content/2016/09/06/css-grid-layout.rst

```diff
@@ -0,0 +1,183 @@
+
+.. code:: scss
+
+  .container {
+    display: flex;
+
+    @supports ( display: grid ) {
+  	   display: grid;
+    }
+  }
+
+You can see a few fallbacks in this CodePen demo:
+
+.. raw:: html
+
+  <p data-height="642" data-theme-id="21914" data-slug-hash="vXBvNE" data-default-tab="css,result" data-user="stacy" data-embed-version="2" class="codepen">See the Pen <a href="http://codepen.io/stacy/pen/vXBvNE/">CSS Grid Layout with float and flexbox fallbacks</a> by Stacy (<a href="http://codepen.io/stacy">@stacy</a>) on <a href="http://codepen.io">CodePen</a>.</p><script async src="//assets.codepen.io/assets/embed/ei.js"></script>
+
+
+Grid introduces new vocabulary and CSS properties
+-------------------------------------------------
+
+New grid terminology includes:
+
+.. line-block::
+
+  **Grid line**
+  The lines that create the grid, separating the grid cells.
+
+  **Grid track**
+  The horizontal or vertical space between two grid lines, often spanning multiple grid cells.
+
+  **Grid cell**
+  A single unit of the grid made from the space between four grid lines.
+
+  **Grid area**
+  A group of space between four grid lines, often containing a group of grid cells. Grid areas can be named in CSS.
+
+
+CSS Grid Layout Properties you would use on a grid container:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+  - display
+  - grid-template-columns
+  - grid-template-rows
+  - grid-template-areas
+  - grid-column-gap
+  - grid-row-gap
+  - grid-gap
+  - justify-items
+  - align-items
+  - justify-content
+  - align-content
+  - grid-auto-columns
+  - grid-auto-rows
+  - grid-auto-flow
+  - grid
+
+
+**CSS Grid Layout Properties you would use on a grid item:**
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+  - grid-column-start
+  - grid-column-end
+  - grid-row-start
+  - grid-row-end
+  - grid-column
+  - grid-row
+  - grid-area
+  - justify-self
+  - align-self
+
+
+`CSS-Tricks posted a guide`_ from Chris House `Chris House`_ that goes in depth into each of these properties.
+
+.. _CSS-Tricks posted a guide: https://css-tricks.com/snippets/css/complete-guide-grid
+.. _Chris House: http://chris.house/blog/a-complete-guide-css-grid-layout/
+
+
+I’ve been creating a few demos in CodePen using Grid and it has been exciting to see the flexibility we will have once this rolls out. I encourage you to start experimenting on your own as well.
+
+
+.. raw:: html
+
+  <p data-height="568" data-theme-id="21914" data-slug-hash="rLyErg" data-default-tab="result" data-user="stacy" data-embed-version="2" class="codepen">See the Pen <a href="http://codepen.io/stacy/pen/rLyErg/">CSS Grid Layout Demo</a> by Stacy (<a href="http://codepen.io/stacy">@stacy</a>) on <a href="http://codepen.io">CodePen</a>.</p><script async src="//assets.codepen.io/assets/embed/ei.js"></script>
```

Comment:
I don't see the layout. What browsers can I see it in? Are there flags I need to turn on? Can you explain some of what you've done to create this? The basics of making it work?